ui/src/app/components/app-router.spec.tsx

```diff
@@ -2,21 +2,12 @@ import '@testing-library/jest-dom';
 
 import * as React from 'react';
 import { MemoryRouter } from 'react-router';
-import { Redirect, Switch } from 'react-router-dom';
+import { Switch } from 'react-router-dom';
+import { Navigate } from 'react-router-dom-v5-compat';
 
 import { render, screen } from '@testing-library/react';
 import { AppRoute, AppRouter, Guard } from 'app/components/app-router';
 
-jest.mock('react-router-dom', () => {
-  const originalModule = jest.requireActual('react-router-dom');
-
-  return {
-    __esModule: true,
-    ...originalModule,
-    BrowserRouter: ({ children }) => <div>{children}</div>,
```

Comment:
don't need this anymore because BrowserRouter is gone.  My understanding is that it was a leftover shim from the Angular days